crates/tabby-common/src/registry.rs

```diff
@@ -54,38 +108,100 @@ pub struct ModelRegistry {
     pub models: Vec<ModelInfo>,
 }
 
+lazy_static! {
+    pub static ref LEGACY_GGML_MODEL_RELATIVE_PATH: String =
+        format!("ggml{}q8_0.v2.gguf", std::path::MAIN_SEPARATOR_STR);
+    pub static ref GGML_MODEL_RELATIVE_PATH: String =
+        format!("ggml{}model.gguf", std::path::MAIN_SEPARATOR_STR);
+    pub static ref GGML_MODEL_PARTITIONED_PREFIX: String = "00001-of-".into();
+}
+
+// model registry tree structure
+// root: ~/.tabby/models/TabbyML
+//
+// fn get_model_root_dir(model_name) -> {root}/{model_name}
+//
+// fn get_model_dir(model_name) -> {root}/{model_name}/ggml
+//
+// fn get_model_path(model_name)
+//   for single model file
+//     -> {root}/{model_name}/ggml/model.gguf
+//   for multiple model files
+//     -> {root}/{model_name}/ggml/{entrypoint}
```

Comment:
make the single / multiple model file name consistent by using `model-000xx-of-000yy` naming routine.

Reply:
the comment is inconsistent with the code, deleted those lines.